src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="page"
+  lang="en"
+>
```

Comment:
The opening `<html>` tag should be followed by a line break, with its attributes indented by 2 spaces for better readability. The closing bracket should also be on the same level as the opening tag.